### DISCUSSION TITLE
Do not require MSAs for the TrueMultimer default mode

### DISCUSSION THREAD

**Discussion by jkosinski** [KosinskiLab/AlphaPulldown#608] [Ideas]
The default TrueMultimer mode now requires MSAs even if PKLs with features exist. Possibly, it could be avoided by extracting the MSA from the PKLs, using that MSA for template search, and then just replacing the template features in the existing PKL files.

**Comment by DimaMolod** [comment]
Broadly harder than it sounds. MonomericObject.make_features() always runs pipeline.process() and then always builds _all_seq pairing features via Uniprot at objects.py (line 140). Downstream multimer assembly expects those MSA-related fields at objects.py (line 709). A narrow “single-sequence only” version is doable, but a full per-protein skip-MSA feature that works cleanly with pulldown pairing is real pipeline surgery.
Moving to ideas.
